### ISSUE TITLE
Sony ZV1-ii (Mark 2) Support

### ISSUE THREAD

**Issue by BabyAteMeDingo** [ma1co/Sony-PMCA-RE#583]
I see the list of supported devices and know the mark 2 ZV1 isnt listed, but will the normal ZV1 work anyway?

Any help is appreciated. Thanks in advance :)

**Comment by kc-2025** [comment]
Yes. It worked for my ZV-1M2.
I followed the method in the following link.
https://www.reddit.com/r/SonyAlpha/comments/18fb0d8/language_unlock_on_newer_alpha_cameras_a6400/?rdt=49568

Good luck!

**Comment by Vhappy86** [comment]
With above procedure do we face camera restart issue when we scroll in language menu?

**Comment by B1ueD0g** [comment]
> 通过上述步骤，当我们在语言菜单中滚动时是否会遇到相机重启问题？

Yes, I have encountered the situation of restarting, and after refreshing, my Japanese machine supports English, but I did not find Chinese
